stacks-signer/src/stacks_client.rs

```diff
@@ -91,6 +94,28 @@ impl StacksClient {
         }
     }
 
+    /// Retrieve the current DKG aggregate public key
+    pub fn get_aggregate_public_key(&self) -> Result<Option<Point>, RPCError> {
```

Comment:
Clarity call should be a function of burn block height only and return an Optional Point (Public Key)